packages/pieces/google-drive/src/lib/action/download-file.ts

```diff
@@ -0,0 +1,87 @@
+import { googleDriveAuth } from '../..';
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { extension, lookup } from 'mime-types';
+
+export const downloadFile = createAction({
+  auth: googleDriveAuth,
+  name: 'download_ggdrive_file',
+  displayName: 'Download file',
+  description: 'Download a selected file from google drive file',
+  props: {
+    fileId: Property.ShortText({
+      displayName: 'File ID',
+      description: 'File ID comming from | New File -> id |',
+      required: true,
+    }),
+    mimeType: Property.ShortText({
+      displayName: 'MIME type',
+      description: 'MIME type comming from | New File -> mimeType |',
+      required: true,
+    }),
```

Comment:
I think it's possible to know mimetype from google while downloading the file, It would be better than asking the user.